src/main/webapp/WEB-INF/tags/instructor/results/sectionPanel.tag

```diff
@@ -36,20 +36,18 @@
             </div>
         </c:when>
         <c:otherwise>
-            <div class="panel-heading ajax_submit">
+            <div class="panel-heading ajax_submit ajax_auto">
```

Comment:
I think we can remove `ajax_submit`, since all panels are always automatically loaded now.